src/ExpressionEvaluator/CSharp/Source/ExpressionCompiler/EvaluationContext.cs

```diff
@@ -550,6 +550,19 @@ internal static ImmutableArray<AssemblyIdentity> GetMissingAssemblyIdentitiesHel
                         }
                     }
                     break;
+                case ErrorCode.ERR_DottedTypeNameNotFoundInNS:
+                    if (arguments.Count == 2)
+                    {
+                        var namespaceName = arguments[0] as string;
+                        var containingNamespace = arguments[1] as NamespaceSymbol;
+                        if (namespaceName != null && (object)containingNamespace != null &&
+                            containingNamespace.ConstituentNamespaces.Select(n => n.ContainingAssembly.Identity).Any(MetadataUtilities.IsWindowsAssemblyIdentity))
```

Comment:
I suspect the fake fusion implementation does not support winmds correctly.  Please open a bug and I'll investigate.